src/pages/workspace/WorkspaceProfilePage.js

```diff
@@ -168,6 +173,15 @@ function WorkspaceProfilePage({policy, currencyList, route}) {
                                     </Text>
                                 </View>
                             </OfflineWithFeedback>
+                            {!readOnly && (
+                                <Button
+                                    style={styles.mt6}
+                                    innerStyles={[StyleUtils.getHeight(variables.workspaceShareButtonHeight), StyleUtils.getWidthStyle(variables.workspaceShareButtonWidth)]}
```

Comment:
Why are we doing a very specific height on this button? We should have standard sizes for our buttons:
- small (28px tall)
- regular/medium/default (40px tall)
- large (52px tall)

You should be using one of these kinds of buttons as the font size changes based on the height, too. 

cc @Expensify/design - what is the intended size here?

Reply:
We decided in Slack awhile ago that Medium (40px) is the intended size for this one.